Havtorn/Source/Engine/ECS/Systems/CameraSystem.cpp

```diff
@@ -55,7 +55,7 @@ namespace Havtorn
 		}
 
 		// === Rotation ===
-		controllerComp->CurrentPitch = UMath::Clamp(controllerComp->CurrentPitch + (CameraRotateInput.X * controllerComp->RotationSpeed * dt), (-UMath::Pi * 0.5f) + 0.01f, (UMath::Pi * 0.5f) - 0.01f);
+		controllerComp->CurrentPitch = UMath::Clamp(controllerComp->CurrentPitch + (CameraRotateInput.X * controllerComp->RotationSpeed * dt), (-180.0f * 0.5f) + 0.01f, (180.0f * 0.5f) - 0.01f);
```

Comment:
Prefer some sort of variable if you're not going to use Pi, otherwise, you might as well change it to -90.0f/90.0f :P